src/autocomplete/StreamAutocomplete.js

```diff
@@ -18,8 +18,10 @@ type Props = {
 class StreamAutocomplete extends PureComponent<Props> {
   props: Props;
 
+  onAutocomplete = (name: string): void => this.props.onAutocomplete(`**${name}**`);
```

Comment:
In a different PR, I decided we should be moving away from this in favor of more explicit but correct code like:

```js
onAutocomplete = (name: string): void => {
  const onAutocomplete = this.props;
  onAutocomplete(`**${name}**`);
}
```
The first is a new eslint rule that warns on destructurng, and the other issue, we should not return the result of `onAutcomplete` even though it is `void` as it might be confusing, why are we returning it? :)

Reply:
@borisyankov Done 👍